frontend/src/lib/components/learningkit/LearningKit.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+	import { Trash2 } from 'lucide-svelte';
+	import ConfirmDialog from '$lib/components/dialogs/ConfirmDialog.svelte';
+	import { _ } from 'svelte-i18n';
+	import { createMutation, useQueryClient } from '@tanstack/svelte-query';
+	import { api } from '$lib/api/apiClient.js';
+	import { deleteLearningKit } from '$lib/api/collections/learningKit.js';
+
+	const client = useQueryClient();
+
+	interface LearningKitProps {
+		title: string;
+		uuid: string;
+		queryKey: string;
+	}
+
+	const { title, uuid, queryKey }: LearningKitProps = $props();
+	let showDeleteDialog = $state(false);
+
+	const deleteKitMutation = createMutation({
+		onSuccess: () => {
+			client.invalidateQueries({ queryKey: [queryKey] });
```

Comment:
I don't think it is appropriate to pass the queryKey from outside and this value should always stay the same: `learning-kit-list` and `latest-learning-kits-list`

If you added a prop purely because of the DRY principle, export reusable const instead. A prop indicates it is meant to be configured, which it shouldnt be in this case